test/cqlpy/test_secondary_index.py

```diff
@@ -193,6 +193,14 @@ def test_create_index_if_not_exists2(cql, test_keyspace, cassandra_bug):
         with pytest.raises(InvalidRequest, match="already exists"):
             cql.execute(f"CREATE INDEX IF NOT EXISTS {index_name} ON {table}(v2)")
 
+# Verify that oversized index names are cleanly rejected  as InvalidRequest
```

Comment:
Below you actually allowed either InvalidRequest or ConfigurationException. Is it because you saw that Cassandra returns one, Scylla returns the other?
I guess that's fine.